submissions/YuraZagor/hooliPopup/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>myPopup</title>
+</head>
+<body>
+	<nav>
+		<ul class="ul--menu">
+			<li class="li--menu"><a href="#">Mail</a></li>
+			<li class="li--menu"><a href="#">Images</a></li>
+
+			<li class="li--menu popup pic" >
+				<label id="popup-label" class="sub-menu" for='checkbox-icon'> <img src="./popup-icons/popup-button.png" alt="popup-label" > </label>
+				<input type="checkbox" id="checkbox-icon" name='checkbox-icon' class='checkbox-icon' alt="popup-checkbox" >
+
+				<section class="outer-popup-container">
+					<div class="inner-container">
+						<ul class="popups--container">
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/blogs.png" alt="articles" class="popup--item"> </a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/apps.png" alt="4-color circle" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/calendar.png" alt="calc" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/chat.png" alt="chat" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/classroom.png" alt="something" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/collection.png" alt="asterisk" class="popup--item"></a></li>	
```

Comment:
As I see in demo the proportions of icons in popup menu  is differs from original. There are [few methods to solve this problem](https://stackoverflow.com/questions/12991351/css-force-image-resize-and-keep-aspect-ratio).

Reply:
man, i really apreciate you trying to help me, yet it reads to me like some hints about Hadron Collider)))  I mean I read and tried those  "[few methods to solve this problem](https://stackoverflow.com/questions/12991351/css-force-image-resize-and-keep-aspect-ratio)."  I just did my best with this task spending so much time (about 10 days) to so little good, so I desided to do the rest of tasks on JS  and then return here later on, if mentors don't tell me to focus on CSS

Reply:
seems now they look better, thanks

Reply:
About aspect ratio better read [this article](https://jakearchibald.com/2022/img-aspect-ratio/)

Reply:
the trick with 	'object-fit: cover;'  helped make video-icon look good, thanks

Reply:
Better use `object-fit: contain` to display full icon)

Reply:
yep, even better, thanks a lot